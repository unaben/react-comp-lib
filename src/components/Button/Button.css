.button {
    font-size: 1rem;
    padding: .5em 1em;
    cursor: pointer;
    color: wheat;
    background-color: steelblue;
}