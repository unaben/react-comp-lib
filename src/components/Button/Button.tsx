import React,{FC} from 'react'
import './Button.css'

interface ButtonProps{
    label:string
}

const Button:FC<ButtonProps> = ({label}) => {
  return (
    <button className='button'>{label}</button>
  )
}

export default Button