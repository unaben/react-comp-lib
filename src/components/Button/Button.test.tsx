import React from 'react'
import {render} from '@testing-library/react'

import Button from './Button'

describe('Button component', () => { 
    it('should render properly', () => {
        const {container} = render(<Button label='Hi' />)
        expect(container).toMatchSnapshot()
    })
 })