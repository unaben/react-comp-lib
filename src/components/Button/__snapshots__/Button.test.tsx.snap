// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component should render properly 1`] = `
<div>
  <button
    class="button"
  >
    Hi
  </button>
</div>
`;
